adapters/sovrn/sovrn.go

```diff
@@ -75,71 +116,54 @@ func addHeaderIfNonEmpty(headers http.Header, headerName string, headerValue str
 	}
 }
 
-func (s *SovrnAdapter) MakeBids(internalRequest *openrtb2.BidRequest, externalRequest *adapters.RequestData, response *adapters.ResponseData) (*adapters.BidderResponse, []error) {
-	if response.StatusCode == http.StatusNoContent {
+func (s *SovrnAdapter) MakeBids(request *openrtb2.BidRequest, requestData *adapters.RequestData, responseData *adapters.ResponseData) (*adapters.BidderResponse, []error) {
```

Comment:
**Nitpick:** The names `requestData` and `responseData ` don't really convey the proper context here IMHO. How about using, `bidderRequest` and `bidderResponse`, respectively?

Reply:
Sure. 😄